spec/services/bulk_upload/lettings/year2023/row_parser_spec.rb

```diff
@@ -30,7 +30,8 @@
   end
 
   before do
-    allow(FormHandler.instance).to receive(:lettings_in_crossover_period?).and_return(true)
+    Timecop.freeze(now)
```

Comment:
I don't think there's a way round time travel in this case - and given it's a 2023 specific parser that seems fine.

Reply:
I'm assuming we'll remove 2023 specific things shortly after the collection close

Reply:
Yeah, I expect so - figured it was worth this level of keeping it in until we do, just in case.